File: Subprogramas/Parametros Variaveis/parametros_variaveis.cs
//C Sharp (C#
using System;

class Program {
    static void PrintNumbers(params int[] numbers) {
        foreach (int number in numbers) {
            Console.WriteLine(number);
        }
    }


    static void Main() {
		// Chama o método com um número variável de parâmetros
        PrintNumbers(1, 2, 3, 4, 5);  
    }
}

File: Subprogramas/Parametros Variaveis/parametros_variaveis.pl
# Perl
use strict;
use warnings;

sub print_numbers {
    my @numbers = @_;
    foreach my $number (@numbers) {
        print "$number\n";
    }
}

# Chama a função com um número variável de parâmetros
print_numbers(1, 2, 3, 4, 5);
